css/jstree-glpi.css

```diff
@@ -329,8 +329,6 @@
   opacity: 0.8;
   filter: url("data:image/svg+xml;utf8,<svg xmlns=\'http://www.w3.org/2000/svg\'><filter id=\'jstree-grayscale\'><feColorMatrix type=\'matrix\' values=\'0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0 0 0 1 0\'/></filter></svg>#jstree-grayscale");
   /* Firefox 10+ */
-  filter: gray;
-  /* IE6-9 */
   -webkit-filter: grayscale(100%);
```

Comment:
According to MDN, `filter: grayscale(100%);` can be used for Firefox. `-webkit` must be kept for edge.

Reply:
Be warn that on this case, the developper write code and the line below the comments ;-)
Else IE6-9 have the webkit rendering engine 😨

Reply:
Yes, I've understand that :D

So, `filter: url()` was for Firefox, but this does not seems needed right now.